### ISSUE TITLE
IR Sensor Bar Turn on and off

### ISSUE THREAD

**Issue by wizardkitty** [vanilla-wiiu/vanilla#51]
One of the small features of the wii u gamepad is its small built in sensor bar, the wii u just sends commands to turn it on or off from what I know.

There are a few games that use this and vwii mode so it may be useful down in the line if somone decides to make hardware or use a pi (or simillar) with gpio to control 2 IR lights.

**Comment by wizardkitty** [comment]
> You would need to attach a sensor bar to the top of, lets say a Steam Deck, in order for this to work.

Indeed, However some devices like the steam deck will need a extra attachment for the other features like amiibo and the camera. Since some games like wii sports club use this feature however it would make some games impossible to play with this method. Its honestly more of a extra since not many games use this feature.

**Comment by Cart1416** [comment]
I don't think we need this, you could just plug in a usb sensor bar and have it always on

**Comment by wizardkitty** [comment]
> I don't think we need this, you could just plug in a usb sensor bar and have it always on

A usb sensor bar is actually too big, this is mainly just a note of a feature of what the game pad has.

If somone wants to make a new wii u gamepad based off of this, this feature could become useful for just toggling a gpio pin for power of 2 ir leds as to avoid unnecessary battery drain.
